<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Evan Teran</author>
 <class>DialogBinaryString</class>
 <widget class="QDialog" name="DialogBinaryString">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Binary String</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="BinaryString" name="binaryString">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="chkSkipNoAccess">
     <property name="text">
      <string>Skip Regions With No Access Rights</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="chkCaseSensitive">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Case Sensitive</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="chkAlignment">
     <property name="text">
      <string>Show Results With This Address Alignment</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cmbAlignment">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>2 Byte Alignment</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 Byte Alignment</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8 Byte Alignment</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>99</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnFind">
       <property name="text">
        <string>&amp;Find</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BinaryString</class>
   <extends>QFrame</extends>
   <header>BinaryString.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>chkSkipNoAccess</tabstop>
  <tabstop>chkCaseSensitive</tabstop>
  <tabstop>chkAlignment</tabstop>
  <tabstop>cmbAlignment</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>btnHelp</tabstop>
  <tabstop>btnFind</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogBinaryString</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
